src/custodia/store/encgen.py

```diff
@@ -23,13 +23,13 @@ class EncryptedOverlay(CSStore):
             auto-generate key file if missing?
         master_enctype (default: A256CBC_HS512)
             JWE algorithm name
-        secret_protection (default: 'encrypt')
-            Determine the kind of protection used to save keys.
+        secret_protection (default: 'encrypt'):
+            Determine the kind of protection used to save keys:
             - 'encrypt': this is the classic method (backwards compatible)
             - 'pinning': this adds a protected header with the key name as
-                         as aad data, to prevent key swapping in the db
+            add data, to prevent key swapping in the db
```

Comment:
Fixing an unexpected indentation raised in the previous pull request: https://travis-ci.org/latchset/custodia/jobs/233391402